source/extensions/common/wasm/context.cc

```diff
@@ -1106,6 +1106,15 @@ void Context::scriptLog(spdlog::level::level_enum level, absl::string_view messa
 // Connection
 bool Context::isSsl() { return decoder_callbacks_->connection()->ssl() != nullptr; }
 
+// Tracing
+void Context::activeSpanSetTag(absl::string_view key, absl::string_view value) {
+  if (!decoder_callbacks_) {
+    return;
+  }
+  Tracing::Span& span = decoder_callbacks_->activeSpan();
```

Comment:
Have you considered a version of this function that uses `encoder_callbacks_`, maybe as a backup?  I ask because I don't know if `decoder_callbacks_` is valid in the context of the HTTP encoder.  Do you have.an example of this working within the `onResponseBody()` of a Context subclass?